site/soc/software/charts/osh/openstack-compute-kit/neutron.yaml

```diff
@@ -40,34 +34,134 @@ data:
+            kill_keepalived: KillFilter, root, keepalived, -HUP, -15, -9
+            
+            # l3 agent to delete floatingip's conntrack state
+            conntrack: CommandFilter, conntrack, root
+            
+            # keepalived state change monitor
+            keepalived_state_change: CommandFilter, neutron-keepalived-state-change, root
+            # The following filters are used to kill the keepalived state change monitor.
+            # Since the monitor runs as a Python script, the system reports that the
+            # command of the process to be killed is python.
+            # TODO(mlavalle) These kill filters will be updated once we come up with a
+            # mechanism to kill using the name of the script being executed by Python
+            kill_keepalived_monitor_py: KillFilter, root, python, -15
+            kill_keepalived_monitor_py27: KillFilter, root, python2.7, -15
+            kill_keepalived_monitor_py3: KillFilter, root, python3, -15
+            kill_keepalived_monitor_py35: KillFilter, root, python3.5, -15
+            kill_keepalived_monitor_py36: KillFilter, root, python3.6, -15
+            kill_keepalived_monitor_py37: KillFilter, root, python3.7, -15
+        dhcp:
+          content: |
+
+            # neutron-rootwrap command filters for nodes on which neutron is
+            # expected to control network
+            #
+            # This file should be owned by (and only-writeable by) the root user
+            
+            # format seems to be
+            # cmd-name: filter-name, raw-command, user, args
+            
+            [Filters]
+            
+            # dhcp-agent
+            dnsmasq: CommandFilter, dnsmasq, root
+            # dhcp-agent uses kill as well, that's handled by the generic KillFilter
+            # it looks like these are the only signals needed, per
+            # neutron/agent/linux/dhcp.py
+            kill_dnsmasq: KillFilter, root, /sbin/dnsmasq, -9, -HUP, -15
+            kill_dnsmasq_usr: KillFilter, root, /usr/sbin/dnsmasq, -9, -HUP, -15
+            
+            ovs-vsctl: CommandFilter, ovs-vsctl, root
+            mm-ctl: CommandFilter, mm-ctl, root
+            dhcp_release: CommandFilter, dhcp_release, root
+            dhcp_release6: CommandFilter, dhcp_release6, root
+            
+            # haproxy
+            haproxy: RegExpFilter, haproxy, root, haproxy, -f, .*
+            kill_haproxy: KillFilter, root, haproxy, -15, -9, -HUP
+            # RHEL invocation of the metadata proxy will report /usr/bin/python
+            # TODO(dalvarez): Remove kill_metadata* filters in Q release since
+            # neutron-ns-metadata-proxy is now replaced by haproxy. We keep them for now
+            # for the migration process
+            kill_metadata: KillFilter, root, python, -9
+            kill_metadata7: KillFilter, root, python2.7, -9
+            kill_metadata35: KillFilter, root, python3.5, -9
```

Comment:
Same here, do we need python3 or python3.6 ?

Reply:
yes, added 3.6 line